controllers/offer.js

```diff
@@ -44,10 +44,19 @@ const deleteOffer = async (req, res) => {
   res.status(204).end()
 }
 
+const getPriceValues = async (req, res) => {
```

Comment:
maybe it should be called priceRange or priceMinMax, not clear what values mean here